db/seeds.rb

```diff
@@ -1,7 +1,32 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'faker'
+
+ # Create User
+user = User.new(
+    email:  "admin@example.com",
+    password:  "helloworld",
+    password_confirmation: "helloworld"
+  )
+user.skip_confirmation!
+user.save!
+
+# Create a list
+1.times do
+  @user = User.last
```

Comment:
No need for @instance variable here, just use `user` that was assigned on line 4.